decidim-accountability/config/locales/en.yml

```diff
@@ -306,18 +306,25 @@ en:
           root_commentable_url: The URL of the resource that ties to this comment
           user_group: The name of the user group that made this comment (if any)
         results:
+          address: The address of the project (if any)
+          children_count: The number of child results
+          comments_count: The number of comments that this result has
           component: The component that the result belongs to
           created_at: The date when the result was created
           description: The description of the result
           end_date: The date when the result ends execution and it is finished
           id: The unique identifier of the result
+          latitude: The latitude of the project in case it has a physical location
+          longitude: The longitude of the project in case it has a physical location
```

Comment:
```suggestion
          latitude: The latitude of the result in case it has a physical location
          longitude: The longitude of the result in case it has a physical location
```